docs-source/usersguide/application.rst

```diff
@@ -609,6 +622,44 @@ File                               Parameters
 :file:`charmm36/tip5pew.xml`       TIP5P-Ew water model\ :cite:`Rick2004` and ions
 =================================  ============================================
 
+The file :file:`charmm36.xml` bundles everything but the water and ions into a single
+file.  In most cases, you can simply include that file, plus one of the water models,
+such as :file:`charmm36/tip4pew.xml` for the TIP4P-Ew water model\ :cite:`Horn2004`:
+::
+
+    forcefield = ForceField('charmm36.xml', 'charmm36/tip4pew.xml')
+
+.. warning:: Drude polarizable sites and lone pairs are not yet supported
```

Comment:
The `charmm36` conversion doesn't contain Drude oscillators or lone pairs (except on waters), however. I'll clarify this distinction.

BTW, it would be great if you could contribute the Drude site reading support to ParmEd!